notebooks/240-efficientdet-object-detection/README.md

```diff
@@ -0,0 +1,20 @@
+# EfficientDet object-detection with OpenVINO
+
+![Result](https://user-images.githubusercontent.com/71766106/226086430-a7e3cdc4-1f99-4c46-89f9-60dcbadea44a.png)
+
+Research Paper : [EfficientDet: Scalable and Efficient Object Detection](https://openaccess.thecvf.com/content_CVPR_2020/papers/Tan_EfficientDet_Scalable_and_Efficient_Object_Detection_CVPR_2020_paper.pdf)\
+Official GitHub : [EfficientDet](https://github.com/google/automl/tree/master/efficientdet)
+
+[EfficientDets](https://github.com/google/automl/tree/master/efficientdet) are a family of object detection models, which achieve state-of-the-art 55.1mAP on COCO test-dev, yet being 4x - 9x smaller and using 13x - 42x fewer FLOPs than previous detectors. EfficientDets also run 2x - 4x faster on GPU, and 5x - 11x faster on CPU than other detectors
```

Comment:
It would be helpful to include a brief summary of what the notebook/tutorial demonstrates, following the guidelines for readme files. Providing an overview of the notebook content will improve its accessibility and help users determine if it meets their needs. Thank you for considering this suggestion.